doc/environment.yml

```diff
@@ -12,7 +12,7 @@ dependencies:
   - pip:
     - jupyter-book
     # duckdb example
-    - duckdb
+    - duckdb>=0.7.1
```

Comment:
@edublancas do we need to update the changelog for this?

Reply:
no need